apps/native-component-list/src/screens/FontScreen.tsx

```diff
@@ -1,25 +1,10 @@
+import FontAwesome5 from '@expo/vector-icons/FontAwesome5';
 import Ionicons from '@expo/vector-icons/Ionicons';
-import { Platform, ScrollView, Text, View } from 'react-native';
+import { Platform, ScrollView, StyleSheet, Text, View } from 'react-native';
 
 export default function FontScreen() {
   return (
     <ScrollView style={{ flex: 1 }}>
-      <View
-        style={{
-          paddingVertical: 10,
-          paddingHorizontal: 15,
-          flexDirection: 'row',
-          justifyContent: 'space-between',
-          flex: 1,
-        }}>
-        <Ionicons name="logo-facebook" size={25} />
-        <Ionicons name="logo-apple" size={25} />
-        <Ionicons name="logo-amazon" size={25} />
-        <Ionicons name="logo-npm" size={25} />
-        <Ionicons name="logo-google" size={25} />
-        <Ionicons name="alarm" size={25} />
-      </View>
-
       <View style={{ paddingVertical: 10, paddingHorizontal: 15, flex: 1 }}>
         <Text style={{ fontFamily: 'space-mono', fontSize: 16 }}>
```

Comment:
we might want to also add a font that we name something totally unrelated like "A-Custom-Font-Name" to verify that we don't regress in that functionality